app/src/main/res/layout/activity_main.xml

```diff
@@ -5,15 +5,164 @@
     android:id="@+id/main"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@color/white"
     tools:context=".MainActivity">
 
+    <View
+        android:id="@+id/background_my_profile"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@drawable/background"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHeight_percent="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline_start"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_percent="0.05" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline_end"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_percent="0.95" />
+
     <TextView
+        android:id="@+id/text_settings"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
+        style="@style/TextAppearance.AppCompat.White24"
+        android:text="@string/settings"
+        app:layout_constraintBottom_toBottomOf="@id/background_my_profile"
+        app:layout_constraintStart_toStartOf="@id/guideline_start"
+        app:layout_constraintTop_toTopOf="@id/background_my_profile"
+        app:layout_constraintVertical_bias="0.08" />
```

Comment:
There is no need to use separate bias for constraint this widget. On design, widgets text "Settings' and button "Log out" are placed in group and align vertical by center vertically of this group. This group has margin 16.dp at top, start and end. So you need put "Settings" and "Log out" to this group and constraint this group instead of widgets individually. It can be done using LinearLayout or chains, or ConstraintLayout Flow.

Reply:
done